packages/template/vite-typescript/package.json

```diff
@@ -11,9 +11,6 @@
   "license": "MIT",
   "main": "dist/ViteTypeScriptTemplate.js",
   "typings": "dist/ViteTypeScriptTemplate.d.ts",
-  "scripts": {
-    "test": "mocha --config ../../../.mocharc.js test/**/*_spec_slow.ts"
-  },
   "engines": {
     "node": ">= 16.4.0"
```

Comment:
<img width="422" alt="image" src="https://github.com/user-attachments/assets/216b3c1f-08aa-4097-8723-6a9abb800d80" />

vitest@2.1.6 depends on node18+. However, the Vite template currently uses vite@5 and does not force node to be upgraded to version 18; this may be something that needs to be faced in the near future, because Vite@6 has been released for a while and it also requires node18.
At least I am leaving a message here to attract the attention of **forgers**.

Reply:
Thanks for the heads-up @caoxiemeihao. We're currently looking at upgrading all Ecosystem projects to Node 22, after which we can finally bump up the supported major version range for Electron Forge!

Reply:
Also as a note: since `vitest` is only used in CI/testing and we've already only been testing against Node 18, this doesn't block testing/release.